framework/CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 Yii Framework 2 Change Log
 ==========================
 
+2.0.44 under development
+------------------------
+
+- Bug #18813: Fix returned value for empty string, passing stringable object is allow in `yii\helpers\BaseJson::decode()` (WinterSilence)
```

Comment:
```suggestion
- Enh #18813: Allow passing stringable object to `yii\helpers\BaseJson::decode()` (WinterSilence)
```

Reply:
@samdark Stringable object are already supported due to `(string)` cast. This PR only changes error message for non-stringable objects (to less specific - previously you know that you passed object to `Json::decode()`, after this PR you get generic "Invalid JSON" error without any clue what is wrong).

Reply:
True, makes more sense, thanks @rob006 for pointing that out.